websites/R/REYFM/presence.ts

```diff
@@ -213,6 +228,8 @@ presence.on("UpdateData", async () => {
 
     showFormat3 = true;
 
+    presenceData.buttons = [{ label: "Listen along!", url: document.URL }];
```

Comment:
You sure it will not redirect to main page in any way?

Reply:
This one is always in a /gaming or a /channel thingy